app/assets/javascripts/discourse/tests/acceptance/category-new-test.js

```diff
@@ -30,21 +30,21 @@ acceptance("Category New", function (needs) {
     );
 
     assert.strictEqual(
-      queryAll(".edit-category-title h2").text(),
+      query(".edit-category-title h2").innerText,
       I18n.t("category.edit_dialog_title", {
         categoryName: "testing",
       })
     );
 
     await click(".edit-category-security a");
     assert.ok(
-      queryAll("button.edit-permission"),
+      exists(".permission-row button.reply-toggle"),
```

Comment:
I'm guessing this is just a better selector, yeah?

Reply:
Yeah, better - this one actually exists 🤣

`queryAll()` always returns a (truthy) jQ object, so doing `assert.ok(queryAll())` does not check anything at all 😄

Reply:
Thanks for the details!

*shakes fists at jQuery*

I don't have any more concerns :+1: